RELEASE_NOTES.md

```diff
@@ -10,7 +10,7 @@
 
 ## New Features
 
-<!-- Here goes the main new features and examples or instructions on how to use them -->
+* We now provide the `DispatchRunnerActor` class, a class to manage actors based on incoming dispatches.
```

Comment:
Given this description, `DispatchManagingActor` doesn't sound like a bad option either. But again, bike shedding, I think it is only marginally better.